### PR TITLE
DOC-7252 New aggregation methods

### PR DESCRIPTION
**Pull request by paperclypse**
I'd love a developmental edit on this one. I'm not sure these concepts are as clear as they could be.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx [hunk 1]
```diff
@@ -471,6 +471,43 @@ Not every field listed in this glossary is required for every condition type. Th
     * Plugin conditions
   </Collapser>
 
+  <Collapser
+    id="signal_aggregation_delay"
+    title="signal[aggregation_window]"
+  >
+    The amount of time before aggregated data is evaluated. Default is **2 minutes**.
+
+    Used with Event flow and Cadence aggregation methods.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx:480]: think this should be sentence case, e.g., 'Used with event flow and cadence aggregation methods.' Or if there's a need to really emphasize them as types, maybe

**Review by zuluecho9 — DISMISSED** [review]

**Review by HenryTech — COMMENTED** [review]
With regards to usage data, and setting up alerts on it:

Recommend Event Timer, since that data will come in late, but it will only be one data point, and we'd rather the aggregation happen very soon after that data point arrives, instead of setting an arbitrary lag.

**Review by HenryTech — COMMENTED** [review]
For Creating NRQL conditions for policies, we should specify that they really only need to set delay that high if they're using the cadence method -- the other methods will very likely not need that much delay (depending on the data shape).

**Review by HenryTech — COMMENTED** [review]
We should not remove references to aggregation window and its definition, since this will be a vital part of the new system.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts.mdx [hunk 2]
```diff
@@ -166,7 +166,7 @@ Here are some user-related query examples. For details on how users are counted,
 To help [manage your billable data](/docs/telemetry-data-platform/get-data-new-relic/manage-data/manage-your-data), you can set alerts to notify you of unexpected increases in usage. Learn how to create alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
 
 <Callout variant="caution">
-  When creating alert conditions, you should set the [Evaluation offset](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions#evaluation-offset) value to 60 minutes or your conditions may not trigger.
+  When creating alert conditions, you should set the [delay/timer](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions#delay-timer) value to 60 minutes or your conditions may not trigger.
```
Comment (HenryTech) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts.mdx:169]: They will only need to set the delay this high if they are using Cadence, in which case it'll be called Delay, not Timer.
We recommend using Event Timer with this use-case, though, and you can get by with setting the Timer to 30 seconds.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 3]
```diff
@@ -347,9 +347,9 @@ Here are some common use cases for NRQL alert conditions. These queries will wor
   </Collapser>
 </CollapserGroup>
 
-## Alert conditions and query order of operations
+## NRQL conditions and query order of operations [#query-order]
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:350]: In a doc about NRQL conditions, we can simply use conditions.

[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:350]: But I don't want to clean up this doc at this point. That can happen at some later time. I don't have a month of Sundays :D

**Review by HenryTech — COMMENTED** [review]
Made most changes myself, left one question for Skilburn

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 4]
```diff
@@ -495,7 +495,7 @@ Here are some tips for creating and using a NRQL condition:
       </td>
 
       <td>
-        As with all alert conditions, NRQL conditions evaluate one single minute at a time. The implicit `SINCE ... UNTIL` clause specifying which minute to evaluate is controlled by your [**Evaluation offset**](#time) setting. Since very recent data may be incomplete, you may want to query data from 3 minutes ago or longer, especially for:
+        As with all alert conditions, NRQL conditions evaluate one single minute at a time. The implicit `SINCE ... UNTIL` clause specifying which minute to evaluate is controlled by your [**delay/timer**](#time) setting. Since very recent data may be incomplete, you may want to query data from 3 minutes ago or longer, especially for:
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:498]: I took at stab at it.

**Pull request merged**